index.html

```diff
@@ -454,7 +453,7 @@ <h2>Syntax Tokens and Keywords</h2>
       <dt class="changed"><code>@nest</code></dt><dd class="changed">Collects a set of <a>nested properties</a> within
         a <a>node object</a>.</dd>
       <dt class="changed"><code>@none</code></dt><dd class="changed">Used as an index value
-        in an id map, language map, type map or elsewhere where a dictionary is
+        in an <a>id map</a>, <a>language map</a>, <a>type map</a> or elsewhere where a dictionary is
```

Comment:
Doc is mostly using oxford comma.  Add one here.

Reply:
... and the W3C style guides also ask for Oxford commas...:-)

Reply:
I have a friend with an Oxford comma tatoo for situations just like this one. 😁 We should add one.

Reply:
How do you tattoo an oxford comma?

Reply:
Ink and a needle. Same as normal tatoos. 😉